app/plugin.go

```diff
@@ -377,7 +379,7 @@ func (a *App) InitPlugins(pluginPath, webappPath string) {
 			if _, err := a.installPlugin(bytes.NewReader(tarball), true); err != nil {
 				l4g.Error("failed to install prepackaged plugin: " + err.Error())
 			}
-			if _, ok := a.Config().PluginSettings.PluginStates[id]; !ok {
+			if _, ok := a.Config().PluginSettings.PluginStates[id]; !ok && id != "zoom" {
```

Comment:
The zoom plugin shouldn't be activated until it's configured. The jira plugin needs to be activated immediately so that users using the built-in one

IMO the special case is actually for jira but since there's only two pre-packaged plugins this is fine for now

Reply:
Agreed. I think we'll probably want to designate the plugins as enabled or disabled by default in the `prepackagedPlugins` map instead of hard-coding one or more plugin ids here, but this is fine for now.